### ISSUE TITLE
Blank pages served after fully configuring.

### ISSUE THREAD

**Issue by westerncanada** [Qrome/marquee-scroller#40]
Good afternoon,

Up until the point where I added all of the API keys to the web interface, I was able to load the main page, and make changes, however now that I've added the news and weather API keys to the web interface, I am no longer able to load the page. 

Running some diagnostics with Curl show: 
`curl: (56) Illegal or missing hexadecimal sequence in chunked-encoding`

But other times nothing loads.
The LED display is working, loading the time, weather and news.
I'm running a version pulled this morning, on a Node MCU esp8266

I tried to add the settings I want to change (news to cbc-news) to the config.h however I think that is ignored for the web stored values after setup.

Very cool project, cannot wait to see if MQTT will ever be added in.

Thank you

**Comment by Qrome** [comment]
As mentioned in the Readme.md -- changing the Settings.h does not do anything after the initial load.  Those are defaults for the first time loading or when you do a full reset.

Can  you please share what version you have loaded?   The current version is 2.2.

**Comment by westerncanada** [comment]
Yeah, I was only using the Settings.h because the web interface would not load for me, so for now I'm making changes to the Settings.h, and calling the /systemreset URL to make changes, its not pretty but does the job.

Confirming this is on version 2.2

I'll download it again, and wipe the NodeMCU to see if it does anything different a bit later.

**Comment by Qrome** [comment]
This latest version of 2.2 had many changes to resolve the memory issues that typically cause the blank pages to start showing up.   I am running the Wemos D1 Mini (quite a few of them) and the latest version 2.2 seems to have resolved the partial page loading issue.  I am wondering if the NodeMCU you have has some different memory limitations.   Unfortunately the web page serving is String intensive -- and there is a struggle to try and keep the code workable at the same time.

Currently I am not having the blank or partial page loading for the Wemos D1 Mini (EPS8266).   I plan on ordering a NodeMCU anyway and try and have a look at it.

**Comment by westerncanada** [comment]
That's a good point, I've got a couple of Wemos D1 Mini Pros here, I might give one a go.  Saves from having to hack up the STL to fit the longer board for the (very nice) case you designed.

The NodeMCU's I'm using are v2 because they fit on breadboard, unlike the v3 that is way too wide and doesn't fit on breadboard at all.

This is what they [look like](https://www.amazon.ca/Kuman-Internet-Development-wireless-controller/dp/B06VV39XD8) they can be had much cheaper from aliexpress if you're good with waiting for shipping.

**Comment by Sterft** [comment]
I have the same issue, though it disappears when I set the news source to not be Reuters strangely enough. After that the pages load just fine. And I think it fixes another issue I had, where it would be stuck in a weather parsing error after some time. 
Using [this board](https://www.aliexpress.com/item/ESP-12E-WeMos-D1-WiFi-uno-based-ESP8266-shield-for-arduino-Compatible/32821901911.html
)

Changed the news source by going to this:
http://192.168.x.x/savenews?displaynews=on&newsApiKey=xxx&newssource=time

**Comment by Qrome** [comment]
If you run into the memory issues, just reboot it for now.
